server/repositories/offender.js

```diff
@@ -74,6 +90,10 @@ function offenderRepository(httpClient) {
     return httpClient.get(`${endpoint}?${query.join('&')}`);
   }
 
+  function getPrisonDetailsFor(prisonId) {
+    return httpClient.get(`${getBaseUrlFrom(config)}/agencies/${[prisonId]}`);
+  }
+
```

Comment:
Should this live in the offender repository? This doesn't match the API structure, and feels unrelated to the other functions in this file. Do we have a `prisons` repository?

Reply:
Correct! I've decided to refactor this in to the PrisonApiRepository to be clearer with our intent